molecule/resources/playbooks/playbook.yml

```diff
@@ -9,3 +9,7 @@
     - name: execute role
       include_role:
         name: stone-payments.squid
+      vars:
+        squid_custom_configs:
+          - "cache_mem 128 MB"
+        squid_diskcache: "aufs {{ squid_coredumpdir }} 1000 16 256"
```

Comment:
This makes absolutely no sense. Don't ever mix a folder being served to third-parties with coredumps.